homeassistant/components/shelly/light.py

```diff
@@ -472,6 +469,29 @@ class RpcShellyLight(RpcShellyLightBase):
     _attr_supported_features = LightEntityFeature.TRANSITION
 
 
+class RpcShellyCctLight(RpcShellyLightBase):
```

Comment:
```suggestion
class RpcShellyCCTLight(RpcShellyLightBase):
```

Reply:
But maybe we keep it `Cct` because `RGB` is `Rgb`

Reply:
The class name is `RpcShellyCctLight`, I think you're looking at an older commit.

Reply:
I was thinking CCT should be caps but Rgb isn’t so at least it’s consistent

Reply:
I used `Rgb` for since to make it more readable (`RgbLight`), I don't mind we align all if you think `CCT` & `RGB` is better